bin/tests/bootstrap-integration.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/bats
+
+lib_dir="/usr/local/lib"
+
+load "$lib_dir/bats-support/load.bash"
+load "$lib_dir/bats-assert/load.bash"
+load "$lib_dir/bats-file/load.bash"
+
+setup () {
+    test_temp_dir="$(temp_make --prefix "otomi-values-")"
+    export ENV_DIR="$test_temp_dir"
+}
+
+teardown () {
+    temp_del "$test_temp_dir"
+    unset ENV_DIR
+}
+
+@test "executing bootstrap.sh should pass with env folder" {
+    git init "$ENV_DIR"
+    bin/bootstrap.sh
+    assert_success
+}
+
+@test "executing bootstrap.sh should fail without env folder" {
+    run bin/bootstrap.sh
+    assert_failure
+}
+
+@test "executing bootstrap.sh multiple times should pass" {
+    git init "$test_temp_dir"
+    bin/bootstrap.sh 
+    run bin/bootstrap.sh
```

Comment:
I'm curious. Why is the second one invoked with `run`?

Reply:
From the docs:

```
When not to use run

In some cases, using run is redundant and results in a longer and less readable code. Here are a few examples.

In case you only need to check the command succeeded, it is better to not use run, since
run command args ...
echo "$output"
[ "$status" -eq 0 ]
is equivalent to

command args ...
```

So the first run is actually not testing for completion but the second time it's run the exit code should be captured.